### PR TITLE
Upgrade to Glob 4.4.0

### PR DESCRIPTION
**Pull request by codydaig**
It also simplifies getGlobbedFiles

### CODE REVIEW

**Comment by ilanbiala** [comment]
@lirantal I'm not sure about this one, can you take a look?

File: config/config.js [hunk 1]
```diff
@@ -18,38 +18,17 @@ module.exports = _.extend(
...
-
 	// The output array
 	var output = [];
 
-	// If glob pattern is array so we use each pattern in a recursive way, otherwise we use glob 
-	if (_.isArray(globPatterns)) {
-		globPatterns.forEach(function(globPattern) {
-			output = _.union(output, _this.getGlobbedFiles(globPattern, removeRoot));
-		});
-	} else if (_.isString(globPatterns)) {
```
Comment (ilanbiala) [inline config/config.js]: What does this logic do that is now handled by `glob`?

Comment (ilanbiala) [inline config/config.js]: @codydaig can you clear up my confusion here?

**Comment by simison** [comment]
BTW this PL has spaces instead of tabs.

**Comment by ilanbiala** [comment]
Yeah @codydaig please use tabs for indentation.

**Comment by ilanbiala** [comment]
This is already out of date, If anyone would like to submit a PR to update Glob to 5.0.0, that would be much appreciated!

**Comment by adamwalz** [comment]
@ilanbiala Done. #462

**Pull request closed without merging**